main/src/com/google/refine/expr/functions/strings/ReplaceEach.java

```diff
@@ -45,7 +55,7 @@ public Object call(Properties bindings, Object[] args) {
                     }
 
                     // replace each occurrence of search with corresponding element in replace
-                    return StringUtils.replaceEachRepeatedly(str, search, replace);
+                    return StringUtils.replaceEachRepeatedly((String) args[0], search, replace);
```

Comment:
This is going to collide with `com.google.refine.util.StringUtils`, so let's switch to the fully qualified name here, since it's the only occurrence.
```suggestion
                    return org.apache.commons.lang.StringUtils.replaceEachRepeatedly((String) args[0], search, replace);
```